src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, Actions) => {
+      state.books.push(Actions.payload);
+    },
+    removeBook: (state, { payload }) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
- [ ] [Optional] Linters are to make our codebase cleaner instead of disabling it try fixing the error :+1: